organization/principles/en.md

```diff
@@ -0,0 +1,183 @@
+> important part of the way that PyGreece community operates, they don't have any form
+> legal force yet.
+
+- The name of the community is "PyGreece Community", abbreviated "PyGreece".
+
+- The name of the legal entity responsible for "PyGreece Community" is "PyGreece AMKE".
+
+- PyGreece AMKE is a civil society actor (Civil Non-Profit Company) created by Greek
+  Python community members to serve the mission of PyGreece community. The mission of
+  PyGreece AMKE can be found [here](../about/en.md#mission).
+
+- The organization detals of PyGreece AMKE are [here](../about/en.md#organization-details)
+
+- Membership is open and free to individuals who wish to actively engage in implementing
+  the PyGreece community mission.
+
+- To become a member, the applicant must send an email to the board of the PyGreece AMKE.
+
+<!-- TODO: Provide a form for member registration -->
+
+- The formal decision making bodies of PyGreece community are:
+
+  - The General Assembly.
+  - The Board of Directors.
+
+- The General Assembly is the highest decision making body of PyGreece community. It meets
+  in a regular annual meeting and in extra sessions when called by a PyGreece community
+  member. The General Assembly runs independent from the PyGreece AMKE, apart from
+  selecting the next board.
+
+- All decisions of the General Assembly are made by simple majority among the voting
+  members. Members may vote by proxy or submit written votes in specific issues to the
+  board. All written votes must be declared by the administrators of PyGreece AMKE at the
+  beginning of the meeting. If there is a draw between two proposals a second vote shall
+  be made and so on.
+
+- An extra meeting of the General Assembly must be held within 28 days of the request.
+  Members should be notified at least 5 days prior to the date of the GA (via discord
+  and/or email).
+
+- The board of directors is responsible to select an independent host & the format of the
+  GA with the agreement of the GA requester.
+
+- The board handles all day-to-day business of the PyGreece AMKE. It is allowed to enter
+  contracts for PyGreece AMKE and handle any issues of the PyGreece community that have
+  not been otherwise regulated in the bylaws or by the General Assembly.
+
+- The administrators of the board have the authority to sign for PyGreece AMKE.
+
+- A member may at any time terminate the membership by informing the board in writing.
+
+- A member that acts against the interests of PyGreece community or PyGreece AMKE may be
+  expelled by the full majority decision of the board of directors. In case this member is
+  a director, the director considered to be expelled is excluded from the vote and the
+  decision needs the full majority of the rest of the board to pass. A member may be
+  expelled from the General Assembly of PyGreece community too, at an Assembly meeting by
+  a regular vote with simple majority.
```

Comment:
The distinction between full majority and simple majority is a bit confusing here. We should maybe use simple majority for the GA and supermajority for the BoD.

Reply:
addressed here [a5e481f](https://github.com/pygreece/community/pull/3/commits/a5e481f23781c65dcbebeafb62ba461690e6d699)